src/components/Header.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
+const logo = require('../assets/images/planet.png');
+
```

Comment:
- Kindly use the ES6 module instead of the commonjs module.`import logo from ....`

Reply:
Done :+1: